### PR TITLE
Release network microsoft.network official 2023 06 01

### PR DESCRIPTION
**Pull request by Joanna-Yang-Art**
# ARM (Control Plane) API Specification Update Pull Request 

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

diagram

[1] [ARM review queue] (for **merge** queues, see [4])  
The PRs are processed by time opened, ascending. Your PR may show up on 2nd or later page. 
If you addressed Step 1 from the diagram and your PR is not showing up in the queue, ensure the label `ARMChangesRequested` 
is removed from your PR. This should cause the label `WaitForARMFeedback` to be added.  
[2] https://aka.ms/azsdk/support/specreview-channel  
[3] [List of SDK breaking changes approvers] in pinned Teams announcement  
[4] [public repo merge queue], [private repo merge queue] (for **ARM review** queue, [1])

If you need further help with anything, see `Getting help` section below.

## Purpose of this PR

What's the purpose of this PR? Check all that apply. This is **mandatory**!

  - [ ] New API version. (Such PR should have been generated with [OpenAPI Hub](https://aka.ms/openapihub)).
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix swagger quality issues in S360.
  - [ ] Other, please clarify:
    - _edit this with your clarification_

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [ ] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data plane related specifications.
- [ ] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can proceed to Step 2, "ARM Review", for this PR.

### Breaking changes review (Step 1)

- If the automation determines you have breaking changes, i.e. Step 1 from the diagram applies to you,
  you must follow the [breaking changes process].  
  **IMPORTANT** This applies even if:
  - The tool fails while it shouldn't, e.g. due to runtime exception, or incorrect detection of breaking changes.
  - You believe there is no need for you to request breaking change approval, for any reason. 
    Such claims must be reviewed, and the process is the same.

### ARM API changes review (Step 2)

- If this PR is in purview of ARM review then automation will add the `ARMReview` label.
- If you want to force ARM review, add the label yourself.
- Proceed according to the diagram at the top of this comment.

## Viewing API changes

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

## Suppressing failures

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[Swagger-Suppression-Process](https://aka.ms/azsdk/pr-suppressions) 
to get approval.

## Getting help

- For general PR approval workflow, see the diagram at the top of this comment.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

[ARM review queue]: https://github.com/search?q=org%3AAzure+is%3Apr+is%3Aopen+label%3AWaitForARMFeedback+-label%3AIDCDevDiv++draft%3Afalse+sort%3Acreated-asc+&type=pullrequests
[List of SDK breaking changes approvers]: https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1689115217750?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1689115217750&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1689115217750
[public repo merge queue]: https://github.com/Azure/azure-rest-api-specs/pulls?q=is%3Aopen+is%3Apr+label%3AMergeRequested+draft%3Afalse+sort%3Acreated-asc
[private repo merge queue]: https://github.com/Azure/azure-rest-api-specs-pr/pulls?q=is%3Aopen+is%3Apr+label%3AMergeRequested+-label%3AApproved-OkToMerge+draft%3Afalse+sort%3Acreated-asc
[breaking changes process]: https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/launchingproductbreakingchanges#breaking-change-process-1

### CODE REVIEW

**Comment by ms-henglu** [comment]
Hi @Joanna-Yang-Art ,

Please check and fix the lint errors: https://github.com/Azure/azure-rest-api-specs/pull/26218/checks?check_run_id=18060584680

**Comment by Joanna-Yang-Art** [comment]
@ms-henglu 
Something wrong on this Avocado test for a long time and it didn't get fixed, this test kept failing for such release and we have to skip it each time. Could you help us unblock?
Attached previous discussion history for record
image

**Comment by rkmanda** [comment]
> @ms-henglu Something wrong on this Avocado test for a long time and it didn't get fixed, this test kept failing for such release and we have to skip it each time. Could you help us unblock? Attached previous discussion history for record image

The ask is to fix the swagger lintdiff errors here - https://github.com/Azure/azure-rest-api-specs/pull/26218/checks?check_run_id=18133490258

All errors except the FATAL errors must be fixed.

**Comment by msftsijielin** [comment]
> > @ms-henglu Something wrong on this Avocado test for a long time and it didn't get fixed, this test kept failing for such release and we have to skip it each time. Could you help us unblock? Attached previous discussion history for record image
> 
> The ask is to fix the swagger lintdiff errors here - https://github.com/Azure/azure-rest-api-specs/pull/26218/checks?check_run_id=18133490258
> 
> All errors except the FATAL errors must be fixed.

Hello @rkmanda, I checked all 5 no fatal errors on bastionhost.json file. Basically, the change was to add an old endpoint back to the swagger which we accidentally missed in previous check-in long time ago. Therefore, the change is a copy-paste of existing endpoint with very limited modification. I think these lint errors are due to some new lint rules. If I have to address them then our existing endpoints will be impacted as well. What's the simplest way to unblock Joanna's PR in this case?

**Comment by msftsijielin** [comment]
Hello @rkmanda please check the PR for lint errors fix and suppression: https://github.com/Azure/azure-rest-api-specs/pull/26494

File: specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/virtualWan.json
Comment (mikekistler) [inline specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/virtualWan.json:9156]:
```suggestion
      "type": "object",
      "properties": {
```

File: specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/network.json
Comment (mikekistler) [inline specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/network.json:199]:
```suggestion
      "type": "object",
      "properties": {
```

**Review by mikekistler — COMMENTED** [review]
I left some suggestions that should fix some of the breaking change issues.

**Comment by mikekistler** [comment]
Breaking changes previously reviewed and approved in #25962

**Comment by zizw123** [comment]
Approving the suppression from last PR

**Review by scbedd — COMMENTED** [review]
Responded over mail as well. The ARM reviewers have final say. If ARM Review signs off, I'll sign off model validation failures. These look like real errors.

**Comment by Joanna-Yang-Art** [comment]
> Responded over mail as well. The ARM reviewers have final say. If ARM Review signs off, I'll sign off model validation failures. These look like real errors.

Thanks @scomtott for sign off, yes we already got ARM sign off and don't think it is a blocking issue for this release. This PR is a combination of all NRP partner PRs. We will reach out to NRP partner that made this change and see if anything they could do with the error.

**Review by scbedd — APPROVED** [review]

**Comment by zizw123** [comment]
> > Responded over mail as well. The ARM reviewers have final say. If ARM Review signs off, I'll sign off model validation failures. These look like real errors.
> 
> Thanks @scomtott for sign off, yes we already got ARM sign off and don't think it is a blocking issue for this release. This PR is a combination of all NRP partner PRs. We will reach out to NRP partners who made those changes and see if anything they could do with the error or it is something they think of by design and the test need to have an update or ignore it.
> 
> Attached email discussion image
> 
> @rkmanda or whoever help merge PRs, please kindly help us merge this PR

The swagger modelValidation should be fixed, if you only need to update your examples, what is blocking this?

**Comment by Joanna-Yang-Art** [comment]
> > > Responded over mail as well. The ARM reviewers have final say. If ARM Review signs off, I'll sign off model validation failures. These look like real errors.
> > 
> > 
> > Thanks @scomtott for sign off, yes we already got ARM sign off and don't think it is a blocking issue for this release. This PR is a combination of all NRP partner PRs. We will reach out to NRP partners who made those changes and see if anything they could do with the error or it is something they think of by design and the test need to have an update or ignore it.
> > Attached email discussion image
> > @rkmanda or whoever help merge PRs, please kindly help us merge this PR
> 
> The swagger modelValidation should be fixed, if you only need to update your examples, what is blocking this?

Hi @zizw123 The ModelValidation shouldn't block us from merged this PR, we have been seeing the error on every release, before it was not blocked us in Automated merging requirements met.
As I mentioned in PR discussion and in email we will reach out to NRP partners who made those changes and see if anything they could do with the error or it is something they think of by design and the test need to have an update or ignore it. We already reached out to partners regarding this and will provide an update on next release. Could you help us merge the PR, we are in critical path to complete this release now. Also we already got approval by @scbedd. There is nothing else we can do now

**Pull request merged**
